std-bits/table/src/main/java/org/enso/table/data/column/storage/type/StorageType.java

```diff
@@ -1,7 +1,59 @@
 package org.enso.table.data.column.storage.type;
 
+import org.enso.base.polyglot.NumericConverter;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZonedDateTime;
+
 /**
  * Represents an underlying internal storage type that can be mapped to the Value Type that is exposed to users.
  */
 public sealed interface StorageType permits AnyObjectType, BooleanType, DateType, DateTimeType, FloatType, IntegerType, TextType, TimeOfDayType {
+  /**
+   * @return the a common StorageType the two types.
+   */
+  static StorageType findGeneralType(StorageType firstType, StorageType secondType) {
```

Comment:
```suggestion
  /**
   * @return a common StorageType for the two types.
   */
  static StorageType findCommonType(StorageType firstType, StorageType secondType) {
```

The doc comment suggests a better name indeed.